docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -328,7 +458,56 @@ Or to set the column ``my_new_partition_column`` as a partition column on a tabl
 
 The current values of a table's properties can be shown using :doc:`SHOW CREATE TABLE </sql/show-create-table>`.
 
-.. _iceberg-type-mapping:
+COMMENT
+~~~~~~~
+
+The Iceberg connector supports setting comments on the following objects:
+
+- tables
+- views
+- table columns
+
+The ``COMMENT`` option is supported on both the table and
+the table columns for the :doc:`/sql/create-table` operation.
+
+The ``COMMENT`` option is supported for adding table columns
+through the :doc:`/sql/alter-table` operations.
+
+The connector supports the command :doc:`COMMENT </sql/comment>` for setting
+comments on existing entities.
+
+Setting comments on tables or columns is not supported in case that the
+Iceberg connector is using the :ref:`Glue catalog <iceberg-glue-catalog>`.
```

Comment:
I believe we can comment on Glue catalog in Iceberg. 
```sql
trino:ebyhr> create table test (c1 int);
CREATE TABLE

trino:ebyhr> comment on table test is 'foo';
COMMENT

trino:ebyhr> comment on column test.c1 is 'bar';
COMMENT

trino:ebyhr> show create table test;
                                  Create Table
--------------------------------------------------------------------------------
 CREATE TABLE iceberg.ebyhr.test (
    c1 integer COMMENT 'bar'
 )
 COMMENT 'foo'
 WITH (
    format = 'ORC',
    format_version = 2,
    location = 's3://trino-ci-test/ebyhr/test-e0bf94ca85f2491a9ce1866dc005543c'
 )
```

Reply:
You are right.

I was mislead about 

https://github.com/trinodb/trino/blob/2a56ea9208fd73f0ed59cb56945b2c654306296d/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java#L679-L682

On Iceberg however the table and column comments don't land on AWS Glue, but instead in the metadata of the table. I'll remove this false information.